CHANGELOG.md

```diff
@@ -1,10 +1,25 @@
 # Ohai Changelog
 
-## UNRELEASED
-* [pr#724](https://github.com/chef/ohai/pull/724) Improved debug logging for cloud plugins
-* [pr#722](https://github.com/chef/ohai/pull/722) Detect paravirt amazon instances without hint files
-* [pr#714](https://github.com/chef/ohai/pull/714) Detect Azure using the Azure agent and DHCP options
-* [pr#724](https://github.com/chef/ohai/pull/725) Add support for the 2 latest EC2 metadata versions
+## [8.11.0](https://github.com/chef/ohai/tree/8.11.0) (2016-03-02)
+[Full Changelog](https://github.com/chef/ohai/compare/v8.10.0...8.11.0)
+
+**Implemented enhancements:**
+
+- Detect Azure on non-bootstrapped hosts [\#657](https://github.com/chef/ohai/issues/657)
+- chef-client hangs \(ohai discovery hangs\) when a server has a mounted nfs filesystem while the remote host does not respond [\#537](https://github.com/chef/ohai/issues/537)
```

Comment:
I added the label to exclude this one so you'll want to regenerate the changelog now for this PR.